package.json

```diff
@@ -58,25 +63,29 @@
     "sucrase": "^3.35.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.24.4",
-    "@babel/eslint-parser": "^7.25.7",
-    "@babel/plugin-syntax-import-assertions": "^7.25.7",
-    "@eslint/js": "^9.11.1",
-    "@ls-lint/ls-lint": "^1.10.0",
+    "@babel/core": "^7.28.5",
+    "@babel/eslint-parser": "^7.28.5",
+    "@babel/plugin-syntax-import-assertions": "^7.27.1",
+    "@eslint/js": "^9.39.1",
+    "@ls-lint/ls-lint": "^2.3.1",
     "@mapbox/rehype-prism": "^0.8.0",
     "@types/mocha": "^10.0.10",
     "@types/node": "^22.13.4",
     "c8": "^7.11.2",
     "chai": "^4.3.6",
     "concurrently": "^7.1.0",
-    "eslint": "^9.11.1",
-    "eslint-plugin-no-only-tests": "^2.6.0",
+    "eslint": "^9.39.1",
+    "eslint-config-prettier": "^10.1.8",
+    "eslint-plugin-no-only-tests": "^3.3.0",
     "globals": "^15.10.0",
     "http-server": "^14.1.0",
+    "husky": "^9.1.7",
     "jsdom": "^19.0.0",
+    "lint-staged": "^16.2.6",
     "livereload": "^0.9.3",
     "mocha": "^9.2.2",
     "nodemon": "^2.0.15",
+    "prettier": "3.6.2",
```

Comment:
can we make sure to add the `^` here

Reply:
The prettier docs recommend saving the exact verision:

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.

https://prettier.io/docs/install/

Reply:
So I looked it up, and yeah the lock file will prevent it from changing, but if you ever need to regenerate the lock file for whatever reason then it may use a newer version within the semver range. And with Prettier that could mean an even a minor/patch version update leads to 100+ files being reformatted unintentionally.

Anyhow I'll put in the `^`, just thought I'd explain the warning.